client/lcd/swagger-ui/swagger.yaml

```diff
@@ -1578,7 +1580,316 @@ paths:
           description: Found no voting parameters
         500:
           description: Internal Server Error
-
+  /distribution/delegators/{delegatorAddr}/rewards:
+    parameters:
+    - in: path
+      name: delegatorAddr
+      description: Bech32 AccAddress of Delegator
+      required: true
+      type: string
+    get:
+      summary: Get the total rewards balance from all delegations
+      description: Get the sum of all the rewards earned by delegations by a single delegator
+      produces:
+      - application/json
+      tags:
+      - ICS24
+      responses:
+        200:
+          description: OK
+          schema:
+            type: array
+            items:
+              $ref: "#/definitions/Coin"
+        400:
+          description: Invalid delegator address
+        500:
+          description: Internal Server Error
+    post:
```

Comment:
How about a `/distribution/delegators/{delegatorAddr}/rewards/withdraw/`? We are not `POST`ing an entity to the rewards.

Reply:
but then you're also not posting a withdrawal and it falls under the same argument